src/SolutionTemplate/UnoSolutionTemplate.WinUI.net6/Wasm/manifest.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Adding a manifest file tends to change the way the have behaves with regards to caching. I think we should be adding an option in the new project experience, as well as in the `dotnet new`.

Reply:
See [here](https://platform.uno/docs/articles/uno-development/debugging-templates.html).

Reply:
Addressed this in the next commit.